TreeTracker/UI/Views/View Trees/ViewTreesViewModel.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ViewTreesViewModel.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 11/7/20.
+//  Copyright © 2020 Greenstand. All rights reserved.
+//
+
+import Foundation
+
+protocol ViewTreesViewModelViewDelegate: class {
+    func viewTreesViewModel(_ viewTreesViewModel: ViewTreesViewModel, didReceiveError error: Error)
+    func viewTreesViewModel(_ viewTreesViewModel: ViewTreesViewModel, didUpdateTrees trees: [Tree])
+}
+
+class ViewTreesViewModel {
+
+    struct TreeCountData {
```

Comment:
Are we using `TreeCountData` anywhere in the ViewModel? We should remove it if we don't need it.

Reply:
okay